apps/src/code-studio/pd/workshop_survey/Demographics.jsx

```diff
@@ -1,133 +0,0 @@
-import React from 'react';
-import {FormGroup} from 'react-bootstrap'; // eslint-disable-line no-restricted-imports
-import FormComponent from '../form_components/FormComponent';
-
-const LABELS = {
```

Comment:
I think the sprawling nature of these deletes make sense. The only thing that makes me wary when we're deleting strings like this is that they seem to represent something conceptually useful. Do you know if we have demographic categories like this stored elsewhere?

Reply:
Looking through the categories mapped to each question, some are used elsewhere and have the same categories (such as gender and race), some are generic number or yes/no questions (gradesTaught), but there are 2 that have specific categorical answers that I'm not seeing elsewhere:

```
highest_education: [
        "High school diploma",
        "Associate's degree",
        "Some college",
        "Bachelor's degree",
        "Master's degree or higher",
      ],

      degree_field: [
        "Art, Music, or Foreign Language",
        "Business",
        "Computer Science",
        "Education",
        "Engineering",
        "English, Language Arts, Drama, or Social Sciences",
        "Mathematics",
        "Science",
        "Other",
      ],
```

Since this is all in the professional development space (and directory), I think the closest-related file these could go in could be `application_constants.rb` since they have a `COMMON_OPTIONS` section that seems to fit the profile (if it's okay to add these in even though they aren't currently being used (could also comment them out))?

Reply:
Sounds good! Thank you for the insight, I've notified them [here](https://codedotorg.slack.com/archives/C0SUN2W3D/p1704740132199179).